### DISCUSSION TITLE
Unable to use the Hana Bank app

### DISCUSSION THREAD

**Discussion by ksnpu01** [List-KR/community#20] [General]
I am currently using the Hana Bank app (mbp.hanabank.com). However, there was a period of two days last week when it didn't work properly. The bank informed me that it was due to the use of private DNS and asked me to disable it. So, I disabled it and was able to use the app. Then, on Friday, they said the issue was resolved, and I started using private DNS again. However, today (6/27), it is not working once again. It's frustrating that the app I use for convenience is causing inconvenience. I would appreciate it if you could resolve the issue so that I can use it properly again. I hope to continue using the app without any problems. Thank you.

**Comment by piquark6046** [comment]
I heard that an employee of hanabnak's zendesk ticket related to the issue was submitted and is being processed.
@ameshkov will handle the issue.

**Comment by ameshkov** [comment]
Technical details on this issue are here:
https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291

TLDR: this is not AdGuard DNS blocking that domain, but the domain owners preventing AdGuard DNS from resolving that domain.
